arviz/utils.py

```diff
@@ -50,7 +51,53 @@ def _var_names(var_names, data):
     return var_names
 
 
-def conditional_jit(function=None, **kwargs):  # noqa: D202
+class lazy_property:  # pylint: disable=invalid-name
```

Comment:
Is this function really needed? Python does this automatically, right?

Reply:
When you say function @ahartikainen what do you mean? This comment is on a class definition?

Reply:
haha, true

I meant that import numba stuff, but now I read it again, this is loading the wrapper once?

Reply:
Yeah -- if you import arviz and immediately disable numba, this will never even try to import it.

Reply:
There's a tradeoff the first time you call the function, though: 
- the first numba function used globally will import numba
- the first time each numba-fied function runs, it will get jit-ted first

Note in my screenshot you can see this performance penalty the first time `geweke` is called.